homeassistant/components/home_connect/binary_sensor.py

```diff
@@ -0,0 +1,65 @@
+"""Provides a binary sensor for Home Connect."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+
+from .const import BSH_DOOR_STATE, DOMAIN
+from .entity import HomeConnectEntity
+
+_LOGGER = logging.getLogger(__name__)
+
```

Comment:
We should set `PARELLEL_UPDATES` constant to limit or not limit number of parallel updates for all entities within this platform. We should do this for all platforms.

https://developers.home-assistant.io/docs/integration_fetching_data#request-parallelism

Reply:
Now that all entities have `async_update`, is this still necessary? Of course the SSE threads are parallel, but this is unrelated to `PARELLEL_UPDATES` I suppose.

Reply:
If there's no I/O in the update and we use `async_update` we don't need to set parallel updates. It will default to no limit.

A future PR could refactor and drop the use of `async_update` and instead do the processing inside `_update_callback` and call `self.async_write_ha_state` at the end. This would remove the extra scheduling on the event loop that we don't need.